pkg/backend/openshiftcluster/delete.go

```diff
@@ -96,8 +97,20 @@ func (m *Manager) Delete(ctx context.Context) error {
 		return err
 	}
 
+	for _, rp := range m.doc.OpenShiftCluster.Properties.RegistryProfiles {
+		if rp.Type == api.RegistryTypeACR {
+			err = m.acrtoken.Delete(ctx, &rp)
+			if err != nil {
+				return err
+			}
+		}
+	}
+
 	m.log.Printf("updating billing record with deletion time")
 	_, err = m.billing.MarkForDeletion(ctx, m.doc.ID)
+	if err != nil && strings.Contains(err.Error(), "NotFound: Entity with the specified id does not exist in the system") {
+		return nil
```

Comment:
Should we still log this error to inform the user that the ID couldn't be found, even if we return nil?

Reply:
mmm, maybe i should just remove it? i needed it in dev as the cluster was created without the billing code, i git pulled and then it wouldn't delete (but maybe that will never happen)